CHANGELOG.md

```diff
@@ -68,6 +77,8 @@ This release has an [MSRV][] of 1.75.
 [#42]: https://github.com/linebender/velato/pull/42
 
 [Unreleased]: https://github.com/linebender/velato/compare/v0.3.0...HEAD
+[0.4.0]: https://github.com/linebender/velato/compare/v0.3.1...v0.4.0
+[0.3.1]: https://github.com/linebender/velato/compare/v0.3.0...v0.3.1
```

Comment:
Forgot this link on the last release, including it here.

Reply:
That's why you should do a release as a separate PR and let someone else review it. Might've caught it.